New background for highlighted comments

yanncloud opened the pull request:
- delete old background
- highlight commentContent with editor-primary-100

yanncloud committed:
New background for highlighted comments

- delete old background
- highlight commentContent with editor-primary-100

yanncloud commented:
@elbotho Ich habe vercheckt, wie ich dich als Reviewer hinzufügen kann. 😅 Ich dachte, dass kann ich noch nach dem Pull Request machen.

Entkenntnis commented:
more people joining the frontend 🎉 🎉  :)

elbotho commented on src/components/comments/comment.tsx:
```diff
@@ -80,7 +80,7 @@ export function Comment({
 
     if (commentRef.current) {
       commentRef.current.style.backgroundColor = isHighlight
-        ? '#fdf5e4'
+        ? ''
         : 'transparent'
     }
```
@yanncloud I think this whole block is needed any more now, could you remove it?

yanncloud replied:
of course :)

elbotho commented:
🎉 nicely done so far, I requested one more change and then we can merge it.

yanncloud committed:
Remove unnecessary block

Remove this block:     if (commentRef.current) {
      commentRef.current.style.backgroundColor = isHighlight
        ? ''
        : 'transparent'
    }

Entkenntnis reviewed (approved):
test for vercel integration

elbotho commented:
@yanncloud Thank you! You can now inform Gregor and send him a link to https://de.serlo-staging.dev/discussions where he can review the changes (in some minutes).
